src/main/java/com/coveros/selenified/element/check/Excludes.java

```diff
@@ -0,0 +1,200 @@
+        }
+        return actualClass;
+    }
+
+    /**
+     * Verifies that the element does not contain the provided expected
+     * attribute. If the element isn't present, this will constitute a failure,
+     * same as a mismatch. This information will be logged and recorded, with a
+     * screenshot for traceability and added debugging support.
+     *
+     * @param attribute - the attribute to check for
+     */
+    void attribute(String attribute);
+
+    default String[] checkAttribute(String attribute, double waitFor, double timeTook) {
+        // record the action and get the attributes
+        String[] allAttributes = getAttributes(attribute, "without", waitFor);
+        // record the result
+        if (allAttributes == null || Arrays.asList(allAttributes).contains(attribute)) {
+            getOutputFile().recordActual(getElement().prettyOutputStart() + " contains the attribute of <b>" + attribute + "</b>",
+                    timeTook, Success.FAIL);
+        } else {
+            getOutputFile().recordActual(getElement().prettyOutputStart() + " does not contain the attribute of <b>" + attribute + "</b>" +
+                    ONLYVALUE + Arrays.toString(allAttributes) + "</b>", timeTook, Success.PASS);
+        }
+        return allAttributes;
+    }
+
+    /**
+     * Verifies that the element's text does not contain the provided expected
+     * text. If the element isn't present, this will constitute a failure, same
+     * as a mismatch. This information will be logged and recorded, with a
+     * screenshot for traceability and added debugging support.
+     *
+     * @param expectedValue the expected value of the element
+     */
+    void text(String expectedValue);
+
+    default String checkText(String expectedValue, double waitFor, double timeTook) {
+        // record the action
+        getOutputFile().recordAction(getElement().prettyOutput() + HASNTTEXT + expectedValue + "</b>", waitFor);
+        // check for the object to the present on the page
+        String elementValue = getElement().get().text();
+        // record the result
+        if (elementValue == null || elementValue.contains(expectedValue)) {
+            getOutputFile().recordActual(getElement().prettyOutputStart() + TEXT + elementValue + "</b>", timeTook, Success.FAIL);
+        } else {
+            getOutputFile().recordActual(getElement().prettyOutputStart() + TEXT + elementValue + "</b>", timeTook, Success.PASS);
+        }
+        return elementValue;
+    }
+
+    /**
+     * Verifies that the element's value does not contain the provided expected
+     * value. If the element isn't present or an input, this will constitute a
+     * failure, same as a mismatch. This information will be logged and
+     * recorded, with a screenshot for traceability and added debugging support.
+     *
+     * @param expectedValue the expected value of the element
+     */
+    void value(String expectedValue);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename method "value" to prevent any misunderstanding/clash with field "VALUE" defined in interface "com.coveros.selenified.element.check.Check". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1845)